packages/components/src/drop-zone/style.scss

```diff
@@ -18,11 +18,14 @@
 	}
 
 	&.is-dragging-over-element {
-		background-color: rgba($blue-dark-900, 0.8);
 
 		.components-drop-zone__content {
 			display: block;
 		}
+
+		&.is-dragging-file {
+			background-color: rgba( $blue-dark-900, 0.8 );
```

Comment:
Can you remove the whitespace inside the parenthesis on this line? It's tripping up the tests and causing travis to fail. 
![image](https://user-images.githubusercontent.com/4267290/43979365-807b511e-9cb8-11e8-891c-efe55ea87b14.png)

Reply:
We just had some discussions about the discrepancies between the WPCS whitespace rules in PHP and CSS the other day. I get confused myself 😄